docs-devsite/auth.md

```diff
@@ -1585,6 +1606,8 @@ Promise&lt;void&gt;
 
 Updates the user's phone number.
 
```

Comment:
I'm new to the node.js implementation, but isn't this just updating a field on the user, like updating the password? It doesn't require phone auth to work I'd think. Why isn't this supported?